lib/dicom_net/association.ex

```diff
@@ -17,15 +17,24 @@ defmodule DicomNet.Association do
   alias Dicom.DataElement
   alias DicomNet.Pdu
 
+  # C-Find specific attributes
+  @cfind_success 0x0000
+  @cfind_pending 0xFF00
+
+  # Generic attributes
+  @dataset_present 0x0001
+  @no_dataset_present 0x0101
+  @unable_to_process 0xC001
+
   def start(init_args) do
     GenServer.start(__MODULE__, init_args)
   end
 
   @impl true
-  def init(%{socket: socket, event_listener: event_listener, getresponses: getresponses}) do
+  def init(%{socket: socket, event_listener: event_listener, handlers: handlers}) do
```

Comment:
`event_listener` should now be removed from the APIs of both Endpoint and Association - it's not used anynmore, right?